packages/grafana-ui/src/components/VizTooltip/VizTooltipContainer.tsx

```diff
@@ -25,7 +25,7 @@ export const VizTooltipContainer: React.FC<VizTooltipContainerProps> = ({
   ...otherProps
 }) => {
   const tooltipRef = useRef<HTMLDivElement>(null);
-  const tooltipMeasurementRef = useRef<Dimensions2D>({ width: 0, height: 0 });
+  const [tooltipMeasurement, setTooltipMeasurement] = useState<Dimensions2D>({ width: 0, height: 0 });
```

Comment:
ah, so the ref was 0 because of the layout effect being called before flushing to DOM :(